### ISSUE TITLE
Issue with snap: won't download

### ISSUE THREAD

**Issue by kz6fittycent** [powersj/ubuntu-iso-download#2]
```
james@ratchet:~$ ubuntu-iso-download server bionic --dry-run --debug
Traceback (most recent call last):
  File "/snap/ubuntu-iso-download/65/bin/ubuntu-iso-download", line 11, in <module>
    load_entry_point('ubuntu-iso-download==19.1', 'console_scripts', 'ubuntu-iso-download')()
  File "/snap/ubuntu-iso-download/65/lib/python3.5/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/snap/ubuntu-iso-download/65/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2793, in load_entry_point
    return ep.load()
  File "/snap/ubuntu-iso-download/65/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2411, in load
    return self.resolve()
  File "/snap/ubuntu-iso-download/65/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2417, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/snap/ubuntu-iso-download/65/lib/python3.5/site-packages/ubuntu_iso_download/__main__.py", line 9, in <module>
    from .iso import ISO
  File "/snap/ubuntu-iso-download/65/lib/python3.5/site-packages/ubuntu_iso_download/iso.py", line 25, in <module>
    from ubuntu_release_info import data as UbuntuReleaseInfo
ImportError: No module named 'ubuntu_release_info'
```
Snap is installed as "classic". 

Snap version: 

```
snap info ubuntu-iso-download 
name:      ubuntu-iso-download
summary:   Download the latest Ubuntu ISOs
publisher: Joshua Powers (powersj)
contact:   https://github.com/powersj/ubuntu-iso-download/issues/new
license:   unset
---
  - ubuntu-iso-download
snap-id:      W1SQfejOZPyJGH4xCZIszXJbeuymJVCU
tracking:     stable
refresh-date: today at 23:45 CDT
channels:
  stable:    e7d6227 2019-05-06 (65) 10MB -
  candidate: ↑                            
  beta:      ↑                            
  edge:      e7d6227 2019-03-31 (65) 10MB -
installed:   e7d6227            (65) 10MB -
```

Any ideas?

**Comment by powersj** [comment]
First, thank you very much for the report! I had thought I had tested the edge channel before I released it, but apparently not :\

Hmm for some reason the snap build did not pull in the `ubuntu-release-info` dependency. This is stated in the requirements.txt file. For now I have reverted the version that is released while I figure it out as I am also into other snap build failures now.

**Comment by powersj** [comment]
Fixed in 63550c3 and release 73 (amd64) on snap store
